app/config/FrontendAppConfig.scala

```diff
@@ -43,6 +43,7 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
   lazy val registerUrl: String                  = configuration.get[String]("urls.register")
   lazy val changeOrganisationDetailsUrl: String = s"$registerUrl/change-contact/organisation/details"
   lazy val changeIndividualDetailsUrl: String   = s"$registerUrl/change-contact/individual/details"
+  lazy val addFinancialInstitution: String      = s"$fIManagementUrl/${controllers.addFinancialInstitution.routes.AddFIController.onPageLoad}"
```

Comment:
I don't think the prefix `$fIManagementUrl` is required since the url is on the same service. I believe `controllers.addFinancialInstitution.routes.AddFIController.onPageLoad.url` should be good enough and could even be defined at the call site.